pkg/pkg.bzl

```diff
@@ -76,6 +76,8 @@ def _pkg_tar_impl(ctx):
         "--owner=" + ctx.attr.owner,
         "--owner_name=" + ctx.attr.ownername,
     ]
+    if ctx.executable.compressor:
+        args.append("--compressor=%s %s" % (ctx.executable.compressor.path, ctx.attr.compressor_args))
```

Comment:
This kind of construction worries me.    How does arg parsing know this arg takes several values.
Just make it two distinct args.  compressor, compressor_args, and the second one is quoted as
--compresser_args='%s'

Reply:
These args are interpreted directly by `argparse`, not the shell, so I think quotes would be interpreted as part of the argument. My guess is it does a naive `split('=')[1]`

Reply:
I confirmed that this works correctly and the value of the argument is a space-delimited string.

Reply:
I'm not convinced, because I know of existing bugs in the way we handle passing args to actions on various platforms. But I'll let it slide now, because I'm going to rewrite this method to use the ctx.action.args rather than a string list. So we'll see what happens then.

Reply:
I added an assertion to validate the arg passing.